### DISCUSSION TITLE
How to import Latin-only version of a variable font like Inter

### DISCUSSION THREAD

**Discussion by jasongitmail** [fontsource/fontsource#674] [Q&A] [answered]
I notice [the Inter variable font loads many versions](https://github.com/fontsource/fontsource/blob/main/fonts/google/inter/latin.css) of the font that I actually do not need--Vietnamese, Greek, Cyrillic, etc. 

There doesn't seem to be a way to import only Inter variable for Latin. Is there a way to do this?

**Comment by ayuhito** [comment, marked as answer]
Unfortunately, it is not possible. Although it shouldn't be a significant issue considering the `unicode-range` selector would prevent the unused font files from ever loading.

Some large fonts, such as Noto Sans JP, use [numbered subsets](https://github.com/fontsource/font-files/blob/main/fonts/variable/noto-sans-jp/wght.css) to reduce the size of downloads even more. While we can access an old Google API that can generate entire subsets like this, we cannot do this with variable fonts since Google's newer APIs favour numbered subsets for large fonts.

It is possible for Inter specifically since that isn't split into numbered subsets. But for some larger fonts, it is not possible and the inconsistency would be challenging to work around as it would be confusing to end-users.

**Comment by jasongitmail** [comment]
Thanks for explaining. The unicode-range does take care of the most important bit, which is download size of the font. I could always add the font-face CSS myself to reduce the unused @font-face's declarations from my CSS. Thanks
